### PR TITLE
feat: Add receipt number column for donations

### PR DESCRIPTION
**Pull request by howar31**
# Issue
[go api-開立收據編號欄位](https://app.asana.com/0/1205812314629275/1207537656691262/f)

# Notice
[Schema docs](https://www.notion.so/twreporter/pay_by_prime_donations-b4e2927d173543fdb961c9c69c5b3506?pvs=4)

# Dependency

### CODE REVIEW

File: migrations/000020_add_receipt_no_column.down.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `pay_by_prime_donations`
+DROP COLUMN `receipt_number`;
+
+ALTER TABLE `periodic_donations`
```
Comment (Aylie-Chou) [inline migrations/000020_add_receipt_no_column.down.sql:4]: The new column `receipt_number` would have to add on `pay_by_card_token_donations` table in order to store each payment's receipt number

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
please have a look at comment

File: migrations/000020_add_receipt_no_column.up.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `pay_by_prime_donations`
+ADD COLUMN `receipt_number` varchar(12) DEFAULT NULL;
+
+ALTER TABLE `periodic_donations`
+ADD COLUMN `receipt_number` varchar(12) DEFAULT NULL;
```
Comment (Aylie-Chou) [inline migrations/000020_add_receipt_no_column.up.sql:5]: 12 -> 13 ?

File: migrations/000020_add_receipt_no_column.down.sql (hunk 1 above)
Comment (howar31) [reply migrations/000020_add_receipt_no_column.down.sql]: done

File: migrations/000020_add_receipt_no_column.up.sql (hunk 2 above)
Comment (howar31) [reply migrations/000020_add_receipt_no_column.up.sql]: done

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**